00-programming-fundamentals/get-to-know-your-text-editor.md

```diff
@@ -27,6 +27,27 @@ Beginning in C11, the Ada Developers Academy classroom portion is converting fro
 
 Please note that if there are any mentions of `Atom` or the command `atom .`, that the document is outdated. With a few exceptions, you should be able to think of them interchangeably.
 
+## VS Code Must Live in `Applications`
+
+In macOS systems, these things are true:
+
+- By default, files downloaded from the Internet get put into your Downloads folder, which is a folder that is located at the path `~/Downloads`. You can navigate to the Downloads folder using the Finder application.
+- Certain features in macOS rely on all apps/programs being located in the Applications folder. You can navigate to the Applications folder using the Finder application.
```

Comment:
Maybe mention that Applications should be in Finder's sidebar?  (Assuming that's still true with the current version of macOS.)